shared/profile/edit-avatar/index.desktop.tsx

```diff
@@ -297,6 +299,140 @@ class EditAvatar extends React.Component<Props, State> {
   }
 
   render() {
+    if (flags.teamsRedesign) {
+      return (
+        <Kb.Modal
+          onClose={this.props.onClose}
+          header={{
+            leftButton: this.props.createdTeam ? (
+              <Kb.Icon type="iconfont-arrow-left" onClick={this.props.onClose} />
+            ) : null,
+            rightButton: this.props.createdTeam ? (
+              <Kb.Button
+                label="Skip"
+                mode="Secondary"
+                onClick={() => undefined}
+                style={styles.skipButton}
+                type="Default"
+              />
+            ) : null,
+            title: (
+              <ModalTitle
+                teamname={this.props.teamname ? this.props.teamname : ''}
+                title="Upload an avatar"
+              />
+            ),
+          }}
+          allowOverflow={true}
+          footer={{
+            content: (
+              <Kb.Button
+                fullWidth={true}
+                label="Continue"
+                onClick={this.onSave}
+                disabled={!this.state.hasPreview}
+              />
+            ),
+          }}
+        >
+          {!!this.props.error && (
+            <Kb.Banner color="red">
+              <Kb.BannerParagraph bannerColor="red" content={this.props.error} />
+            </Kb.Banner>
+          )}
+          <Kb.Box
+            className={Styles.classNames({dropping: this.state.dropping})}
+            onDragLeave={this.onDragLeave}
+            onDragOver={this.onDragOver}
+            onDrop={this.onDrop}
+            style={Styles.collapseStyles([
+              styles.container,
+              {
+                paddingTop: this.props.createdTeam ? 0 : Styles.globalMargins.xlarge,
+              },
+            ])}
+            onMouseUp={this.onMouseUp}
+            onMouseDown={this.onMouseDown}
+            onMouseMove={this.onMouseMove}
+          >
+            {this.props.createdTeam && (
+              <Kb.Box style={styles.createdBanner}>
+                <Kb.Text type="BodySmallSemibold" negative={true}>
+                  Hoorah! Your team {this.props.teamname} was created.
```

Comment:
I can't find this banner in the design.

Reply:
https://zpl.io/2pe84qo

Reply:
So this is fine right?

Reply:
Yup!

Reply:
I'm fixing this in my branch, but the problem with this is that this code isn't distinguishing between new team in the wizard and normal just-created-team.